extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -57,6 +63,32 @@ class AmpStickyAd extends AMP.BaseElement {
   buildCallback() {
     this.viewport_ = this.getViewport();
     this.element.classList.add('i-amphtml-sticky-ad-layout');
+
+    // Setting padding-bottom to avoid iPhone home bar
+    if (isExperimentOn(this.win, 'sticky-ad-padding-bottom')) {
+      const experimentInfoMap = /** @type {!Object<string,
+        !../../../src/experiments.ExperimentInfo>} */ ({
+        [[STICKY_AD_PADDING_BOTTOM_EXP.id]]: {
```

Comment:
Nit: You're stringifying an array here
```suggestion
        [STICKY_AD_PADDING_BOTTOM_EXP.id]: {
```

Reply:
Not sure why we do it in different ways, but you are right. Our use has been inconsistent.